src/plugins/chart_expressions/expression_xy/public/components/tooltip/tooltip.scss

```diff
@@ -22,9 +22,14 @@
   }
 }
 
+.detailedTooltip__labelContainer {
+  max-width: $euiSizeXL * 5;
+}
+
```

Comment:
Rather than adding a `max-width` here, could you try removing and instead replacing lines 8–17 above with the following? I believe that giving the table some notion of its boundaries will address the issue. I also removed the redundant `word-wrap` style from this code chunk.

```
  table {
    table-layout: fixed;
    width: 100%;

    td,
    th {
      text-align: left;
      padding: $euiSizeXS;
      overflow-wrap: break-word;
    }
  }
```